reference/forms/types/collection.rst

```diff
@@ -182,7 +182,7 @@ And update the template as follows:
         <ul id="email-fields-list"
             data-prototype="{{ form_widget(form.emails.vars.prototype)|e }}"
             data-widget-tags="{{ '<li></li>'|e }}"
-            data-widget-counter="{{ form.children|length }}">
+            data-widget-counter="{{ form.emails|length }}">
```

Comment:
It is clear to me that the length of `$form->children` is not the same as the length of `$form->children['emails']`, so the fix is correct.